grpc-ballerina/tests/resources/generated-sources/tool_test_data_type_7/helloWorldWithNestedMessage_pb.bal

```diff
@@ -0,0 +1,203 @@
+import ballerina/grpc;
+
+public client class helloWorldWithNestedMessageClient {
+
+    *grpc:AbstractClientEndpoint;
+
+    private grpc:Client grpcClient;
+
+    public isolated function init(string url, *grpc:ClientConfiguration config) returns grpc:Error? {
+        self.grpcClient = check new(url, config);
+        check self.grpcClient.initStub(self, ROOT_DESCRIPTOR, getDescriptorMap());
+    }
+
+    isolated remote function hello(HelloRequest|ContextHelloRequest req) returns (HelloResponse|grpc:Error) {
+        
+        map<string|string[]> headers = {};
+        HelloRequest message;
+        if (req is ContextHelloRequest) {
+            message = req.content;
+            headers = req.headers;
+        } else {
+            message = req;
+        }
+        var payload = check self.grpcClient->executeSimpleRPC("helloWorldWithNestedMessage/hello", message, headers);
+        [anydata, map<string|string[]>][result, _] = payload;
+        
+        return <HelloResponse>result;
+        
+    }
+    isolated remote function helloContext(HelloRequest|ContextHelloRequest req) returns (ContextHelloResponse|grpc:Error) {
+        
+        map<string|string[]> headers = {};
+        HelloRequest message;
+        if (req is ContextHelloRequest) {
+            message = req.content;
+            headers = req.headers;
+        } else {
+            message = req;
+        }
+        var payload = check self.grpcClient->executeSimpleRPC("helloWorldWithNestedMessage/hello", message, headers);
+        [anydata, map<string|string[]>][result, respHeaders] = payload;
+        
+        return {content: <HelloResponse>result, headers: respHeaders};
+    }
+
+    isolated remote function bye(ByeRequest|ContextByeRequest req) returns (ByeResponse|grpc:Error) {
+        
+        map<string|string[]> headers = {};
+        ByeRequest message;
+        if (req is ContextByeRequest) {
+            message = req.content;
+            headers = req.headers;
+        } else {
+            message = req;
+        }
+        var payload = check self.grpcClient->executeSimpleRPC("helloWorldWithNestedMessage/bye", message, headers);
+        [anydata, map<string|string[]>][result, _] = payload;
+        
+        return <ByeResponse>result;
+        
+    }
+    isolated remote function byeContext(ByeRequest|ContextByeRequest req) returns (ContextByeResponse|grpc:Error) {
+        
+        map<string|string[]> headers = {};
+        ByeRequest message;
+        if (req is ContextByeRequest) {
+            message = req.content;
+            headers = req.headers;
+        } else {
+            message = req;
+        }
+        var payload = check self.grpcClient->executeSimpleRPC("helloWorldWithNestedMessage/bye", message, headers);
+        [anydata, map<string|string[]>][result, respHeaders] = payload;
+        
+        return {content: <ByeResponse>result, headers: respHeaders};
+    }
+
+}
+
+
+
+
```

Comment:
Is it possible to remove these extra spaces?

Reply:
Yes, these test resources are the output of `Alpha5`. Will replace with the new output which does not have such extra spaces.

Reply:
Fixed in f623090